.gitignore

```diff
@@ -28,3 +28,5 @@ npm_licenses.tar.bz2
 /.build
 
 /**/node_modules
+
+.idea/
```

Comment:
We do not add editor's gitignore in Prometheus. Please consider using a global .gitignore on your computer.